docs/source/tutorials/e2e_flow.rst

```diff
@@ -522,5 +522,31 @@ The output should look something like this:
 
 And thats it! Try your own prompt!
 
+Uploading your model to the Hugging Face Hub
+--------------------------------------------
+
+Your new model is working great and you want to share it with the world. The easiest way to do this
+is utilizing the ``huggingface-cli`` command, which is integrated with torchtune. Simply point the CLI
+to your finetuned model directory like so:
+
+.. code-block:: bash
+
+    huggingface-cli upload <hf-repo-id> <checkpoint-dir>
+
+The command should output a link to your repository on the Hub. If the repository doesn't exist yet, it will
+be created automatically:
+
+.. code-block:: text
+
+    https://huggingface.co/<hf-repo-id>/tree/main/.
+
+.. note::
+
+    Before uploading, make sure you are authenticated with Hugging Face by running ``huggingface-cli login``.
```

Comment:
FYI, You can also mention the "authentication" section from the quickstart guide (https://huggingface.co/docs/huggingface_hub/quick-start#authentication). And yeah, `huggingface-cli login` is perfectly valid!